Protocol.md

```diff
@@ -134,6 +134,8 @@ When using a transport protocol that does not provide compatible framing, the Fr
 
 __NOTE__: Byte ordering is big endian.
 
+Regardless of whether the Frame Length field is required, all RSocket Frames (Header + Payload) have a size limit of 16,777,215 bytes.
```

Comment:
Thoughts on expanding this to use similar language to other portions of the protocol and adding an additional header?

```suggestion
### Max Frame Size

RSocket Frames (Header + Payload) MUST be limited to 16,777,215 bytes, regardless of whether the utilized transport protocol requires the Frame Length field.
```